src/software/firmware/srcs/pressure_controller.cpp

```diff
@@ -433,16 +435,14 @@ PressureController::pidPatient(int32_t targetPressure, int32_t currentPressure,
     patientLastError = error;
 
     int32_t patientCommand = (PID_PATIENT_KP * error) + patientIntegral
-                             + ((PID_PATIENT_KD * derivative) / 1000);  // calcul de la commande
+                             + ((PID_PATIENT_KD * derivative) / 1000);  // Command computation
 
     int32_t minAperture = m_blower.minAperture();
     int32_t maxAperture = m_blower.maxAperture();
 
-    uint32_t patientAperture =
+    int32_t patientAperture =
```

Comment:
We can move from int32 to uint32. 

The reason is we backported the code from a previous commit, because the current HEAD on the master is messy with the PID.